### PR TITLE
Fix wrong example in doc, sync Python E2E version and adjust E2E

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by Superskyyy** [comment]
Umm looks like Python e2e fails mysteriously.
Is it just me or the CI summary in this repo won't open properly?
image

**Comment by Superskyyy** [comment]
I think I found the problem. 

It should be 

`RUN make setup-test install`

in the Dockerfile, requests is no longer a default setup dependency since end of 2020...

File: oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/k8s/K8SServiceRegistry.java [hunk 1]
```diff
@@ -64,9 +64,9 @@
 
     protected final ServiceNameFormatter serviceNameFormatter;
 
-    private final EnvoyMetricReceiverConfig config;
+    protected final KubernetesNodeRegistry nodeRegistry;
 
-    private final KubernetesNodeRegistry nodeRegistry;
+    private final EnvoyMetricReceiverConfig config;
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/k8s/K8SServiceRegistry.java:69]: Are these expected changes?

File: oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/k8s/KubernetesNodeRegistry.java [hunk 2]
```diff
@@ -42,7 +42,7 @@
 import org.apache.skywalking.oap.server.library.util.StringUtil;
 
 @Slf4j
-final class KubernetesNodeRegistry implements ResourceEventHandler<V1Node> {
+public final class KubernetesNodeRegistry implements ResourceEventHandler<V1Node> {
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/als/k8s/KubernetesNodeRegistry.java:45]: Same here.

**Review by wu-sheng — DISMISSED** [review]

**Comment by kezhenxu94** [comment]
@wu-sheng should be good to merge now

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**